### ISSUE TITLE
Additional Data in .rodare.json

### ISSUE THREAD

**Issue by ax3l** [tobiasfrust/test-rodare#1]
What are additional fields I can put in the `.rodare.json`?

Can I list
- keywords
- grants
- "Related identifiers" (cites ...)

as well? :-) This would be awesome and save me a lot of manual work per release :)

**Comment by tobiashuste** [comment]
Yes, you can add all metadata keys into this file as defined in the metadata scheme given in this link: https://gitlab.hzdr.de/rodare/rodare/blob/master/zenodo/modules/deposit/jsonschemas/deposits/records/legacyrecord.json

You can test all those things in https://rodare-test.hzdr.de from inside the HZDR Network. Just use a random pub_id number there.

I will add some more keys to this example to make it more complete!

**Comment by ax3l** [comment]
Nice, thanks that's super useful!
